lib/Cake/Model/Datasource/DboSource.php

```diff
@@ -367,6 +367,18 @@ public function value($data, $column = null, $null = true) {
 			$column = $this->introspectType($data);
 		}
 
+		if (strpos($column, "enum") === 0) {
+			preg_match("/(enum\()(.*)(\))/", $column, $acceptingValues);
+			if(preg_match("/(enum\()(.*)(\))/",$column, $acceptingValues)){
```

Comment:
Why do you call `preg_match()` twice here? Once should be enough if you capture the matches.

Reply:
> Why do you call `preg_match()` twice here? Once should be enough if you capture the matches.

I used GitHub functionality and I thought it will remove the first one, but not.

Reply:
> Why do you call `preg_match()` twice here? Once should be enough if you capture the matches.

Done